### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by subhashjain010**
First Time Contribution

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+     <!-- ________ Subhash's card START ________  -->
+        <div class="card">
+          <p class="name">Subhash Jain</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@Subhash01010820</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://www.twitter.com/Subhash01010820" target="_blank">@Subhash01010820</a>
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,30 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+               <a href="https://medium.com" target="_blank" title="Second Resource">Medium</a>
+              </li>
+              <li>
+                <a href="https://replit.com" target="_blank" title="Third resource">Replit</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________Subhash's card END ________  -->
```
Comment (TenTraicion) [inline index.html:141]: add a line break after you card

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly do the required changes and commit!

**Comment by subhashjain010** [comment]
> Kindly do the required changes and commit!

done

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@subhashjain010, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**